### PR TITLE
- comparison section with cosmos subsection added

### PR DESCRIPTION
**Pull request by drskalman**
- included in intro.
- label added for Avalability.

### CODE REVIEW

**Comment by Swader** [comment]
I believe this is better suited as an update, where needed, to this: https://wiki.polkadot.network/docs/en/learn-comparisons-cosmos

File: docs/polkadot/Polkadot-OverviewPaper/comparison.tex [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+It is noteworthy that using SPREE modules, Polkadot offers even stronger security than shared security (See Section \ref{sec:spree}. When a parachain signs up for a SPREE module, Polkadot guarantees that certain XCMP messages received by that parachain are being processed by the  pre-defined SPREE module set of code. No similar cross-zone trust framework is offered by Cosmos system.
+
+Another significant difference between Cosmos and Polkadot is that on how blocks are produced and finalized. In Polkadot because all parachain states are strongly connected to relay chain states, parachain can temporarily fork alongside the relay chain. This allows for block production to decouple from finality logic. In this sense, Polkadot blocks can be produced over unfinalized blocks and multi blocks can be finalized at once. One the other hand, Cosmos zone depends on instant finality of Hub states to perform cross-chain openation and therefore delayed finalization halts the cross zone operations.
```
Comment (keorn) [inline docs/polkadot/Polkadot-OverviewPaper/comparison.tex]: Please run through spellcheck: _openation_ -> _operation_

File: docs/polkadot/Polkadot-OverviewPaper/comparison.tex [hunk 2]
```diff
@@ -0,0 +1,11 @@
+\subsection{Comparison with other multi-chain systems}\label{sec:comparison}
+
+\paragraph{Cosmos} 
+
+Much like Polkadot, Cosmos is a system designed to solve blockchain interoperability problem which is fundamental to improve the scalability for the decntralized web. In this sense, there are many similarities between the two systems. Hence, Cosmos consists of components which play similar roles as the sub-components of Polkadot. For example, the Cosmo's Hub resembles the Polkadot Relay Chain, or similar to Polkadot parachains, Cosmos's zones are the blockchains which use the Hub to interact between one another.
```
Comment (lsaether) [inline docs/polkadot/Polkadot-OverviewPaper/comparison.tex]:
```suggestion
Much like Polkadot, Cosmos is a system designed to solve the blockchain interoperability problem which is fundamental to improve the scalability for the decntralized web. In this sense, there are many similarities between the two systems. Hence, Cosmos consists of components which play similar roles as the sub-components of Polkadot. For example, the Cosmo's Hub resembles the Polkadot Relay Chain, or similar to Polkadot parachains, Cosmos's zones are the blockchains which use the Hub to interact between one another. 
```

**Comment by lsaether** [comment]
Looks accurate. Just suggested some rephrasing to the first paragraph to improve clarity.

**Comment by drskalman** [comment]
I  also had @AlistairStewart 's blessing.

**Pull request merged**